src/components/posts/PostsList.vue

```diff
@@ -0,0 +1,140 @@
+<template>
+  <div>
+
+    <v-list
+        class="py-0"
+        outlined
+        rounded
+        width="100%"
+        color="transparent"
+    >
+      <template v-for="(post, index) in posts">
+        <v-list-item
+            :key="index"
+        >
+          <v-list-item-content>
+            <v-card
+                class="pa-5"
+                min-height="130"
+            >
+              {{ post.content }}
+            </v-card>
+          </v-list-item-content>
+          <v-btn
+              class="ma-2"
+              text
+              icon
+              color="blue lighten-2"
+              @click="giveReaction(post.id, 'like')"
```

Comment:
lets simulate a enum

create in some global directory aN object:
```
        const REACTIONS = {
          LIKE: 'like',
          DISLIKE: 'dislike',
          NONE: 'none',
      }
```
then use it like that whenever referring to a reaction type
```suggestion
              @click="giveReaction(post.id, REACTIONS.LIKE)"
```